about_us.html

```diff
@@ -174,113 +205,214 @@
                             src: "./assets/images/avatars/eric.png",
                             name: "Eric Carmichael",
                             title: "Owner, Lead Developer",
-                            description: "Lorem ipsum dolor sit amet.",
+                            description: ["Eric started programming at a young age. At the age of 15, he fell in love with it when hacked his school’s school stock exchange. CKC was founded in 2012 so Eric could have the flexibility to work and take care of his mother’s health. Since then we’ve had the opportunity to work with some of the largest Django shops and run projects from wireframes to millions in funding.",
```

Comment:
I'd like to move this slides array out of our HTML and into its own file that we can import so we don't have a massive text dump in the middle of our Vue file.

Reply:
I moved this array in the latest push, make sure you pull from the branch before you re-push